gcovr/__main__.py

```diff
@@ -61,10 +61,25 @@ def fail_under(covdata, threshold_line, threshold_branch):
         percent_branches = 100.0
 
     if percent < threshold_line and percent_branches < threshold_branch:
+        logger.error(
+            "Failed, not enough line coverage (got: {}, expected min: {})",
+            percent,
+            threshold_line
+        )
         sys.exit(6)
     if percent < threshold_line:
+        logger.error(
+            "Failed, not enough line coverage (got: {}, expected min: {})",
+            percent,
+            threshold_line
+        )
         sys.exit(2)
     if percent_branches < threshold_branch:
+        logger.error(
+            "Failed, not enough branch coverage (got: {}, expected min: {})",
+            percent,
+            threshold_branch
```

Comment:
Should be `percent_branches`, not `branch`.

Could we add some simple tests that check that the messages contain the correct values? The makefile-based tests can't do this, but capturing STDERR in `test_args.py` should work (by providing coverage data via JSON in a tempfile).

Reply:
I'll add a test.